icons/uv.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>uTorrent</title><path d="m0 .1058.0504 11.9496.0403 9.5597c.0055 1.3199 1.08 2.3854 2.4 2.3798l9.5596-.0403 5.9749-.0252.6075-.0026c1.316-.0056 2.3799-1.0963 2.3799-2.4123h1.0946v2.3894L24 23.9042 23.8992.005 12.9056.0513l.0463 9.5245v5.9637h-1.9583L11.04 9.584 10.9936.0594Z"/></svg>
```

Comment:
You have the title here as uTorrent, not uv.

Reply:
Fixed.

Reply:
Would you be open to adding a title-filename check to svglint?

Reply:
We have this. It's just because you're a first-time contributor, some checks will not be run automatically.

Reply:
I ran the lint command locally before I pushed this commit. Didn't show up. Any idea what I might've done wrong?

Ran `npm run lint` once again after changing the title to uTorrent.
This was the output:
```
> simple-icons@13.14.0 lint /home/aniruddha/Personal Projects/simple-icons
> npm run ourlint && npm run prettierlint && npm run jslint && npm run jsonlint && npm run svglint && npm run wslint


> simple-icons@13.14.0 ourlint
> node scripts/lint/ourlint.js


> simple-icons@13.14.0 prettierlint
> npm run prettier -- --check


> simple-icons@13.14.0 prettier
> prettier --cache --ignore-unknown "**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts|svg)" --check

Checking formatting...
All matched files use Prettier code style!

> simple-icons@13.14.0 jslint
> xo

> simple-icons@13.14.0 jsonlint
> node scripts/lint/jsonlint.js


> simple-icons@13.14.0 svglint
> svglint --ci --config svglint.config.mjs icons/*.svg

----------------------------------- Summary -----------------------------------
✓ 3217 valid files.

> simple-icons@13.14.0 wslint
> editorconfig-checker

```
[Screencast from 17-10-24 12:51:00 PM IST.webm](https://github.com/user-attachments/assets/8373580f-7e1c-4c03-b5fe-e7acfdc4cef4)

Reply:
I see. I will file a PR to add this support.

Reply:
I'm willing to contribute as well. Shall I open a PR?